app/i18n/en-us/index.php

```diff
@@ -17,7 +17,7 @@
 		'bugs_reports' => 'Bug reports',	// IGNORE
 		'credits' => 'Credits',	// IGNORE
 		'credits_content' => 'Some design elements come from <a href="http://twitter.github.io/bootstrap/">Bootstrap</a> although FreshRSS doesn’t use this framework. <a href="https://git.gnome.org/browse/gnome-icon-theme-symbolic">Icons</a> come from the <a href="https://www.gnome.org/">GNOME project</a>. <em>Open Sans</em> font police has been created by <a href="https://fonts.google.com/specimen/Open+Sans">Steve Matteson</a>. FreshRSS is based on <a href="https://framagit.org/marienfressinaud/MINZ">Minz</a>, a PHP framework.',	// IGNORE
-		'freshrss_description' => 'FreshRSS is a self-hostable RSS feed aggregator like <a href="http://tontof.net/kriss/feed/">Kriss Feed</a> or <a href="https://github.com/LeedRSS/Leed">Leed</a>. It is light and easy to use, whilst being a powerful and configurable tool.',	// IGNORE
+		'freshrss_description' => 'FreshRSS is a self-hostable RSS feed aggregator that is light and easy to use whilst being highly powerful and configurable.',	// IGNORE
```

Comment:
Btw, whilst is chiefly British but I don't know if that's worth the effort.

Reply:
> FreshRSS is a self-hostable RSS aggregator and reader. It allows you to read and follow several news websites at a glance without the need to browse from one website to another. FreshRSS is lightweight, configurable, and easy to use.

How about this? I pulled the first two sentences from [the manual](https://freshrss.github.io/FreshRSS/en/).

Otherwise: 
> FreshRSS is a self-hostable RSS feed aggregator that is light and easy to use _while_ being highly powerful and configurable.

Reply:
> How about this? I pulled the first two sentences from [the manual](https://freshrss.github.io/FreshRSS/en/).

Yes, I copy edited the manual a bit (fairly superficially, but I smoothed over a fair bit of Frenglish). I completely forgot there even was such a thing as an about page, and that explains my odd feeling that I'd already changed it. ;-)

> Otherwise:

I prefer the above of course. But yes, while would be the usual way to write that for the en-US localization.